### PR TITLE
Use named arguments in the DNS name format

### PR DESCRIPTION
**Pull request by ikitiki**
Instead of using sprintfplaceholders use named arguments

### CODE REVIEW

File: pkg/util/config/util.go [hunk 1]
```diff
@@ -162,3 +165,23 @@ func processField(value string, field reflect.Value) error {
...
 	return nil
 }
+
+func (f *stringTemplate) Decode(value string) error {
+	*f = stringTemplate(value)
+
+	return nil
+}
+
+func (f *stringTemplate) Parse(a ...string) string {
```
Comment (a1exsh) [inline pkg/util/config/util.go]: The function name looks surprising to me, as it does the opposite of "parsing".

Comment (ikitiki) [inline pkg/util/config/util.go]: yeah, you're right.
what do you think about naming it ``Execute`` ?

Comment (a1exsh) [inline pkg/util/config/util.go]: `Format`?

Comment (ikitiki) [inline pkg/util/config/util.go]: ``Format`` sounds ok for me

File: pkg/util/config/config.go [hunk 2]
```diff
@@ -43,19 +43,19 @@ type Config struct {
...
-	Workers            uint32 `name:"workers" default:"4"`
+	Namespace          string         `name:"namespace"`
+	EtcdHost           string         `name:"etcd_host" default:"etcd-client.default.svc.cluster.local:2379"`
+	DockerImage        string         `name:"docker_image" default:"registry.opensource.zalan.do/acid/spiloprivate-9.6:1.2-p4"`
+	ServiceAccountName string         `name:"service_account_name" default:"operator"`
+	DbHostedZone       string         `name:"db_hosted_zone" default:"db.example.com"`
+	EtcdScope          string         `name:"etcd_scope" default:"service"`
+	WALES3Bucket       string         `name:"wal_s3_bucket"`
+	KubeIAMRole        string         `name:"kube_iam_role"`
+	DebugLogging       bool           `name:"debug_logging" default:"true"`
```
Comment (alexeyklyukin) [inline pkg/util/config/config.go]: Did you intentionally change the debug_logging to true?

Comment (ikitiki) [inline pkg/util/config/config.go]: No, it was unintentionally, but I don't mind keeping it true by default

**Comment by alexeyklyukin** [comment]
👍

**Comment by ikitiki** [comment]
👍

**Pull request merged**